packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1126,7 +1332,6 @@ RawFloatingCursorPoint _toTextPoint(FloatingCursorDragState state, Map<String, d
 class TextInput {
   TextInput._() {
     _channel = SystemChannels.textInput;
```

Comment:
Does it still make sense for the `TextInput` to hold the `SystemChannel`?

Reply:
There were a couple of reasons I left it here.
- `TextInput.setChannel()` for testing
- I'm not sure what to do with `TextInput.finishAutofillContext()`, which is calling the method channel directly and bypassing any text input connection. I was tempted to add `TextInputConnection.finishAutofillContext()`, but it would change the behavior so that it would only do something if a client is attached. Any idea what is the desired behavior? Apparently `TextInput.finishAutofillContext` is not implemented on Linux, at least.